File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/comment/uncomment/direct/Java/Python/s466963472_commented.py
import sys

def solve(d):
    ans = 0
    i = 1
    while i * d < 600:
        x = i * d
        ans += x * x * d
        i += 1
    return ans

def main():
    data = list(map(int, sys.stdin.read().split()))
    for d in data:
        print(solve(d))

if __name__ == "__main__":
    main()
# End of Code

File: tasks/translation/output/Qwen/Qwen2.5-Coder-32B-Instruct-AWQ/codenet/cot/Java/Go/s466963472_commented.go
package main

import (
	"fmt"
)

var d int

func read() bool {
	_, err := fmt.Scan(&d)
	return err == nil
}

func solve() int {
	ans := 0
	for i := 1; i*d < 600; i++ {
		x := i * d
		y := x * x
		ans += y * d
	}
	return ans
}

func main() {
	for read() {
		fmt.Println(solve())
	}
}
// End of Code
